puml/fhir_version_timeline.puml

```diff
@@ -34,6 +34,7 @@ footer Zuletzt verändert am %date("dd.MM.yyyy")
 concise "de.gkvsv.eRezeptAbrechnungsdaten" as GKV <<gkvcolor>>
 concise "kbv.ita.erp" as KBV <<kbvcolor>>
 concise "S_KBV_DARREICHUNGSFORM" as DF <<kbvcolor>>
+concise "S_KBV_DMP" as DMP <<kbvcolor>>
```

Comment:
Mit den Schlüsseltabellen und deren impliziter Kennzeichnung mit der “S_*”-Notation ist die Grafik leider nicht ohne weitere Erklärungen zugänglich. Ich schlage vor, die Grafik um eine detaillierte Legende oder einen Erklärungstext zu ergänzen, der auf die jeweiligen Simplifier-Projekte der angezeigten Pakete verweist. Vermutlich genügt eine einfache Tabelle in der folgenden Form:

| Paketname                              | Beschreibung                                           | Simplifier-Projekt                                    |
|----------------------------------------|--------------------------------------------------------|-------------------------------------------------------|
| de.gkvsv.eRezeptAbrechnungsdaten       |                                                        |                                                       |
| kbv.ita.erp                            | eRezept-Projekt der KBV                                | [eRezept](https://simplifier.net/eRezept/)            |
| S_KBV_DARREICHUNGSFORM                 |                                                        |                                                       |
| S_KBV_DMP                              |                                                        |                                                       |
| de.abda.eRezeptAbgabedaten             |                                                        |                                                       |
| de.abda.eRezeptAbgabedatenPKV          |                                                        |                                                       |
| de.gematik.erezept-workflow.r4         | E-Rezept-Workflow-Projekt der gematik                  | [E-Rezept-Workflow](https://simplifier.net/erezept-workflow) |
| de.gematik.erezept-patientenrechnung.r4| E-Rezept-Patientenrechnung - Abrechnungsinformationen des E-Rezeptes für den Patienten - der gematik | [E-Rezept-Patientenrechnung](https://simplifier.net/erezept-patientenrechnung) |